src/terminal/node/terminal-backend-module.ts

```diff
@@ -22,13 +22,20 @@ export default new ContainerModule(bind => {
 @injectable()
 class TerminalExpressContribution implements BackendApplicationContribution {
     private terminals: Map<number, any> = new Map()
-    private logs: string[] = []
+
+    getShellExecutablePath() {
+        if (process.platform === 'win32') {
```

Comment:
I can't find that in the doc https://nodejs.org/api/os.html or in the index.d.ts from node ? where did you see this isWindows exactly?